larq_compute_engine/core/packbits.h

```diff
@@ -6,6 +6,7 @@
 #include <cstring>
 #include <limits>
 
+#include "larq_compute_engine/core/macros.h"
```

Comment:
The current `macros.h` file, which we weren't using anywhere, uses a very bad version of determining endianness, by calling some networking-related function. This not only will not compile on microcontroller platforms, it might also incur actual runtime overhead to do that check. 
I therefore suggest we update `macros.h` by using `FLATBUFFERS_LITTLEENDIAN` or so instead, since all our projects always have flatbuffers as a dependency anyway.
This has nothing to do with this PR by the way, but I realized that adding this include file would break our microcontroller builds.

Reply:
Ah interesting, I thought this was a compile-time constant that would get optimised away. I will update it, thanks :+1: